fiware-region-sanity-tests/commons/fiware_cloud_test_case.py

```diff
@@ -31,8 +31,10 @@
 from novaclient.exceptions import NotFound, ClientException as NovaClientException
 from novaclient.exceptions import ConnectionRefused as NovaConnectionRefused
 from neutronclient.common.exceptions import NeutronClientException
+from swiftclient.exceptions import ClientException
```

Comment:
As with the rest of ClientExceptions, you could use the "as" mechanism and import this as "SwiftClientException"